psse/psse-converter/src/test/resources/SwitchedShunt.xiidm

```diff
@@ -0,0 +1,140 @@
+                <iidm:step r="0.0" x="0.0" g="0.0" b="0.0" rho="0.987004441519987"/>
+            </iidm:ratioTapChanger2>
+            <iidm:ratioTapChanger3 lowTapPosition="0" tapPosition="0" loadTapChangingCapabilities="false">
+                <iidm:step r="0.0" x="0.0" g="0.0" b="0.0" rho="1.0001250156269534"/>
+            </iidm:ratioTapChanger3>
+            <iidm:currentLimits1 permanentLimit="116.62475437630441"/>
+            <iidm:currentLimits2 permanentLimit="6447.0780059508215"/>
+            <iidm:currentLimits3 permanentLimit="10861.401939129835"/>
+        </iidm:threeWindingsTransformer>
+    </iidm:substation>
+    <iidm:substation id="S3">
+        <iidm:voltageLevel id="VL3" nominalV="18.0" topologyKind="BUS_BREAKER">
+            <iidm:busBreakerTopology>
+                <iidm:bus id="B3" name="3           " v="18.36" angle="-37.1199"/>
+            </iidm:busBreakerTopology>
+            <iidm:generator id="B3-G1 " energySource="OTHER" minP="0.0" maxP="890.0" voltageRegulatorOn="true" targetP="800.0" targetV="18.36" targetQ="466.25" bus="B3" connectableBus="B3">
+                <iidm:regulatingTerminal id="B3-SwSH1"/>
+                <iidm:minMaxReactiveLimits minQ="-9900.0" maxQ="9900.0"/>
+            </iidm:generator>
+            <iidm:shunt id="B3-SwSH1" sectionCount="3" voltageRegulatorOn="false" targetV="17.91" targetDeadband="0.17999999999999972" bus="B3" connectableBus="B3">
+                <iidm:shuntNonLinearModel>
+                    <iidm:section b="-0.06172839506172839" g="0.0"/>
+                    <iidm:section b="-0.030864197530864196" g="0.0"/>
+                    <iidm:section b="-0.015432098765432098" g="0.0"/>
+                    <iidm:section b="0.0" g="0.0"/>
+                    <iidm:section b="0.046296296296296294" g="0.0"/>
+                </iidm:shuntNonLinearModel>
+            </iidm:shunt>
+        </iidm:voltageLevel>
+        <iidm:voltageLevel id="VL5" nominalV="500.0" topologyKind="BUS_BREAKER">
+            <iidm:busBreakerTopology>
+                <iidm:bus id="B5" name="5           " v="486.39" angle="-42.3008"/>
+            </iidm:busBreakerTopology>
+            <iidm:load id="B5-L1 " loadType="UNDEFINED" p0="2000.0" q0="100.0" bus="B5" connectableBus="B5"/>
+            <iidm:load id="B5-L2 " loadType="UNDEFINED" p0="0.0" q0="50.0" connectableBus="B5"/>
+        </iidm:voltageLevel>
+        <iidm:twoWindingsTransformer id="T-3-5-1 " r="0.0" x="28.0" g="0.0" b="0.0" ratedU1="18.0" ratedU2="500.0" bus1="B3" connectableBus1="B3" voltageLevelId1="VL3" bus2="B5" connectableBus2="B5" voltageLevelId2="VL5">
+            <iidm:ratioTapChanger lowTapPosition="0" tapPosition="11" targetDeadband="2.700000000000003" loadTapChangingCapabilities="false" regulating="true" targetV="18.45">
+                <iidm:terminalRef id="B3-SwSH1"/>
+                <iidm:step r="0.0" x="0.0" g="10.803324099723" b="10.803324099723" rho="1.0526315789473684"/>
```

Comment:
b and g are equal here? In general, g is close to zero...

Reply:
Here, in the steps of the transformer r, x, g, and b are percentages, the percentages that should be applied to the real values of  r, x, g and b of the transformer for considering the correction after moving the ratio from one end to the other. The g and b percentages are identical as the ratio affects in the same way to both values of the shunt admittance.